src/_includes/head.html

```diff
@@ -28,12 +28,6 @@
   <meta property="og:image" content="{{og_image_path | absolute_url}}">
 
   <link href="https://fonts.googleapis.com/css?family=Google+Sans:400|Roboto:300,400,500|Source+Code+Pro|Material+Icons" rel="stylesheet">
-  {% asset main.css %}
-  {% for css in page.css -%}
-    {% asset '{{css}}' %}
-  {% endfor %}
-  {% asset main.js %}
-  <link rel="canonical" href="{{ page_url }}">
```

Comment:
Were these the lines that you inadvertently removed? They should be restored.

Reply:
Yes, that was some of the lines, not all.

Reply:
Thanks for recovering part of the lines but this is still missing `{% asset main.js %}`. I'll add it back in a followup PR.

Reply:
Done #1306